packages/jupyterlab-chat/src/model.ts

```diff
@@ -349,3 +427,37 @@ export class LabChatModel
 
   private _user: IUser;
 }
+
+/**
+ * The chat context to be sent to the input model.
+ */
+export class LabChatContext extends ChatContext {
```

Comment:
Do we need to provide both `LabChatContext` and `ChatContext`? There are other areas in this branch where we are unsafely casting `IChatContext` objects as `LabChatContext` to access the `users` property defined here. If we require this property, then it doesn't seem to make sense to have the `IChatContext` exclude that property. 

I recommend that the `users` property be added to the `IChatContext` interface to remove the type casts. I also suggest we merge `ChatContext` and `LabChatContext` together, since one interface will only require one implementation. Is this possible? If not, what is the reason for providing 2 implementations in 2 different packages?